src/components/Tooltip/Tooltip.js

```diff
@@ -212,6 +215,7 @@ export default class Tooltip extends Component {
     iconTitle: '',
     triggerText: 'Provide triggerText',
     menuOffset: getMenuOffset,
+    clickToOpen: breakingChangesX ? true : false,
```

Comment:
just a question but would we prefer to follow https://eslint.org/docs/rules/no-unneeded-ternary.html ?

```suggestion
    clickToOpen: !!breakingChangesX
```

but actually since it looks like we are exporting `breakingChangesX` as false we can just set the value of `clickToOpen` to `breakingChangesX` right?

```suggestion
    clickToOpen: breakingChangesX,
```